en/developer_guide/install_with_vagrant.rst

```diff
@@ -69,8 +69,9 @@ Install open-orchestra with composer:
 
 .. code-block:: bash
 
-    $ ./composer.phar create-project open-orchestra/open-orchestra path/to/your/folder -s dev
-    $ ./composer.phar create-project open-orchestra/open-orchestra-front-demo path/to/your/folder -s dev
+    $ ./composer.phar create-project open-orchestra/open-orchestra path/to/your/back/folder -s dev
+    $ ./composer.phar create-project open-orchestra/open-orchestra-front-demo path/to/your/front/folder -s dev
+    $ ./composer.phar create-project open-orchestra/open-orchestra-provision path/to/your/provisionning/folder -s dev
```

Comment:
So we have a problem as the provisionning stuff is no more on the open-orchestra bundle: this doc is a non-sense.

Or it should not be public and instead be placed in the open-orchestra-provision repo.